<?xml version="1.0"?>

<launch>
	<!-- Force sensor driver -->
	<param name="robot_description" textfile="$(find hardware)/urdf/kuka.urdf" />
	<node name="KUKA" pkg="motions" type="motions" output="screen" required="true">
		<rosparam param="toolLink"> "contact_point" </rosparam>
		<rosparam param="freq"> 100.0 </rosparam>
		<!-- Control mode -->
		<rosparam param="controlMode"> "jointVel" </rosparam>
		<!-- jointVel -->
		<rosparam param="jointVel"> [0.01, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01] </rosparam>
	</node>
	<node pkg="robot_state_publisher" type="robot_state_publisher" name="rob_st_pub__" />
</launch>
